dashboard/lti_new.py

```diff
@@ -0,0 +1,198 @@
+import logging, string, random
+
+import django.contrib.auth
+
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.views.decorators.http import require_POST
+from django.views.decorators.csrf import csrf_exempt
+from pylti1p3.contrib.django import DjangoOIDCLogin, DjangoMessageLaunch, DjangoCacheDataStorage
+from pylti1p3.tool_config import ToolConfDict
+from django.contrib.auth.models import User
+from dashboard.common.db_util import canvas_id_to_incremented_id
+from django.core.exceptions import ObjectDoesNotExist
+from dashboard.models import Course, CourseViewOption
+from django.conf import settings
+
+
+logger = logging.getLogger(__name__)
+INSTRUCTOR = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor'
+TA = 'http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistant'
+COURSE_MEMBERSHIP = 'membership'
```

Comment:
As the others are now explicit, make `COURSE_MEMBERSHIP` explicit, too.  We never know whether there will be some other "membership" URIs in the future, ones that we might not want to match.

```suggestion
COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'
```